### PR TITLE
Update New-AppLockerPolicy.md

### PR DESCRIPTION
**Pull request by msbemba**
Added where {$_.path -like '*.exe'} to the example to exclude DLL files from the applocker policy

### CODE REVIEW

**Comment by msbemba** [comment]
Issue# https://github.com/MicrosoftDocs/windows-powershell-docs/issues/1924

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/windows/applocker/New-AppLockerPolicy.md [hunk 1]
```diff
@@ -81,6 +81,8 @@ If only path information is available for a file, then the file is skipped becau
 If the *IgnoreMissingFileInformation* parameter is not specified when file information is missing, then the cmdlet exits because it cannot create the specified rule type.
 After the new AppLocker policy is created, the AppLocker policy of the specified Group Policy Object (GPO) is set.
 The existing AppLocker policy in the specified GPO will be overwritten.
+If *where {$_.path -like '*.exe'}* is not specified it could cause the workstations to enable rule enforcement on DLL files and cause the workstation to not trust Windows DLL files
```
Comment (get-itips) [inline docset/windows/applocker/New-AppLockerPolicy.md:84]: The word _Workstation_ is not used in these articles, instead _computer_ it is, can we use that word?
Also, I think this paragraph should be before the _After the new AppLocker policy_ paragraph

**Comment by get-itips** [comment]
I receive no results either. @msbemba can you kindly share the results of your tests?

**Comment by msbemba** [comment]
@get-itips  and @dariomws  Thank you for the response.  I could not get a result as well but I had no warning as such in the event log. But it makes sense to exclude the DLL. How do we proceed now? As I see snip from you have warning events that don't show up when the cmdlet is used.

Thanks
Sri

**Comment by dariomws** [comment]
@msbemba to be honest, I´m not sure about this. So, I cannot say I´m ok to go.

What do you think @get-itips and @tseward?

**Comment by get-itips** [comment]
I think that we should be sure, before publishing something that does not work.
@msbemba Would it be possible for you to make another test in your lab until you get results ?

**Comment by msbemba** [comment]
> I think that we should be sure, before publishing something that does not work.
> @msbemba Would it be possible for you to make another test in your lab until you get results ?

Thank you @get-itips, I will do that.

**Comment by e0i** [comment]
@msbemba Please let us know whenever the revisions are ready. Thank you.

**Comment by msbemba** [comment]
Thank you @e0i .  @get-itips I see warning on Exe and DLL but nothing shows up in PS. So could this be an issue with Get-AppLockerFileInformation cmdlet?

**Comment by dariomws** [comment]
@msbemba, not sure... I have the same behaviour in my machine.

If we cannot forward this issue to an SME or PM... Maybe we should dismiss it... What do you think?

**Comment by get-itips** [comment]
> @msbemba, not sure... I have the same behaviour in my machine.
> 
> If we cannot forward this issue to an SME or PM... Maybe we should dismiss it... What do you think?

Agree

**Comment by msbemba** [comment]
@e0i The issue can be closed now.

**Pull request closed without merging**